core/lattice_lightsheet_core/ui_core.py

```diff
@@ -6,11 +6,11 @@
 from napari.types import ImageData

```

Comment:
We need to check if these imports initialise napari in any way. 
If so, it can trigger errors related to display/Xvfb.

Reply:
Good catch, my goal is to remove all `napari` imports from this module entirely.